run_functional_tests.sh

```diff
@@ -34,9 +34,16 @@ function run_minio_server() {
     tests/functional/minio server --config-dir tests/functional/.cfg tests/functional/.d{1...4} >tests/functional/minio.log 2>&1 &
 }
 
-run_minio_server
-sleep 3
+if [ -z ${SERVER_ENDPOINT+x} ]; then
+    run_minio_server
+    trap 'pkill -9 minio' INT
```

Comment:
this will kill any minio process owned by the user running in the system. you would need to kill only if the process the script created. you could redirect failure to `/dev/null` to avoid error message.

Reply:
Yeah will bring it back..